src/TheRestOfYourLife/pdf.h

```diff
@@ -13,6 +13,7 @@
 
 #include "rtweekend.h"
 
+#include "hittable_list.h"
```

Comment:
This needs to be added to the text of the book.

Either spin up a new issue, or fix it in a new commit